src/main/java/com/poiji/bind/Poiji.java

```diff
@@ -233,4 +240,34 @@ private static Unmarshaller deserializer(final InputStream inputStream, PoijiExc
             throw new InvalidExcelFileExtension("Invalid file extension (" + excelType + "), excepted .xls or .xlsx");
         }
     }
+
+    /**
+     * Validate that all headers specified via @ExcelCellName annotations are present in the list of header names.
+     * <p>
+     * Validation is only performed if it is set in the PoijiOptions
+     *
+     * @throws HeaderMissingException if one or more headers are missing
+     */
+    public static <T> void validateMandatoryNameColumns(PoijiOptions options,
```

Comment:
```suggestion
    public static <T> void validateMandatoryNameColumns(PoijiOptions options,
```
we might need to add this method to another package or in another class wdyt?

Reply:
The underlying problem: If you add a field to your model referenced by name via @ExcelCellName, all values will be set to null if the column is missing. If the value is allowed to be null, you can't distinguish between the user forgot to add the column and the user wants to set the value to null.

This can lead to unintentional data loss if
a) the user accidentially deletes or renames a column
b) you add columns to your data model after go live.

In our case it's due to backwards compatibility:
We use Poiji to import bulk patches to master data. Now the user has an existing import file and prepared some changes. At some point in time we add a new nullable field that already exists in the api for a longer time. If the user now imports his "old" format without the new colum, Poiji will parse the old file without errors, but the import will set all values of the new field with null override all existing values of the new column.

I will add an example to the docs.

Reply:
I was thinking about renaming the `SheetNameExtractor` to `Helper` or similar and put it there, but I did not find a proper name.